test/common/run-tests

```diff
@@ -344,8 +344,11 @@ def run(opts, image):
     global_machines = []
 
     if not opts.machine:
-        # create serial machines; adjust for few/no serial tests
-        for i in range(min(opts.batches, len(serial_tests))):
+        # Create appropriate number of serial machines; prioritize the nondestructive (serial) tests, to get
+        # them out of the way as fast as possible, then let the destructive (parallel) ones "float around" them
```

Comment:
"float around" is confusing.  I think you mean "start running as soon as a given serial runner is finished" or so?